lib/reqTemplate.js

```diff
@@ -138,7 +140,7 @@ function splitAndPrepareTAsseblyTemplate(templateSpec, options) {
 
                 // TODO: Enable single brace structures outside of URI once
                 // the specs are migrated to double brace.
-                if (!inDoubleBrace && options.isURI) {
+                if (!inDoubleBrace) {
```

Comment:
To the comment above: given that we have https://github.com/wikimedia/restbase/pull/516 and it's almost done, we can resolve that TODO right now and simultaneously merge RB PR.